building-from-source.rst

```diff
@@ -168,14 +172,14 @@ To install these, you can use:
   unzip it, and then pass ``-DPCAP_ROOT_DIR="<path to npcap sdk>"`` to the
   initial CMake invocation for Zeek.
 
+  Note also that the ZeroMQ cluster backend is not yet supported on Windows.
```

Comment:
Think that's mostly because there's no libzmq available with `choco`. Compiling libzmq on Windows by hand would probably work.